### ISSUE TITLE
investigate dropping healpix (Trac #395)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#394]
We currently maintain a fork of healpix (http://healpix.jpl.nasa.gov/). Coordinate w/ Juan Carlos, and investigate dropping this fork in favor of upstream's, and how we want to manage it (I3_PORTS? system libs?)

Migrated from https://code.icecube.wisc.edu/ticket/395
```json
{
    "status": "closed", 
    "changetime": "2015-02-19T19:23:42", 
    "description": "We currently maintain a fork of healpix (http://healpix.jpl.nasa.gov/). Coordinate w/ Juan Carlos, and investigate dropping this fork in favor of upstream's, and how we want to manage it (I3_PORTS? system libs?)", 
    "reporter": "nega", 
    "cc": "olivas, blaufuss", 
    "resolution": "fixed", 
    "_ts": "1424373822557444", 
    "component": "other", 
    "summary": "investigate dropping healpix", 
    "priority": "minor", 
    "keywords": "healpix", 
    "time": "2012-05-17T14:23:41", 
    "milestone": "", 
    "owner": "nega", 
    "type": "task"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20120523T18:22:59: 
* kjmeagher commented:

> Added tool for healpix in r1228/IceTray
> made branch for dst to use new tool in r88620/IceCube and r88621/IceCube


* kjmeagher changed status from "new" to "assigned"
* nega changed _comment0 from:

> Added tool for healpix in [IT1228]
> made branch for dst to use new tool in [88620] and [88621]

to:

> 1423692256822290

**Comment by icecube-trac** [comment]
Trac update at 20150211T22:03:33: 
* olivas changed cc from "olivas blaufuss" to "olivas, blaufuss"
* olivas changed owner from "kjmeagher" to "nega"

**Comment by icecube-trac** [comment]
Trac update at 20150219T19:23:42: 
* nega changed resolution from "" to "fixed"
* nega commented:

> In changeset 129276:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="129276"
> support healpix as tool, not a project
> 
> closes: #394
> ```


* nega changed status from "assigned" to "closed"
